src/lib/util/timeconv.cpp

```diff
@@ -14,47 +14,47 @@
 
 
 namespace util {
-namespace {
 /***************************************************************************
     PROTOTYPES
 ***************************************************************************/
 
-util::ntfs_duration calculate_ntfs_offset();
+static std::chrono::system_clock::duration calculate_system_clock_adjustment();
 
 
 /***************************************************************************
     GLOBAL VARIABLES
 ***************************************************************************/
 
-util::ntfs_duration f_ntfs_offset(calculate_ntfs_offset());
+std::chrono::system_clock::duration system_clock_adjustment(calculate_system_clock_adjustment());
 
 
 /***************************************************************************
     IMPLEMENTATION
 ***************************************************************************/
 
-util::ntfs_duration calculate_ntfs_offset()
+static std::chrono::system_clock::duration calculate_system_clock_adjustment()
 {
 	constexpr auto days_in_year(365);
 	constexpr auto days_in_four_years((days_in_year * 4) + 1);
 	constexpr auto days_in_century((days_in_four_years * 25) - 1);
 	constexpr auto days_in_four_centuries((days_in_century * 4) + 1);
 
-	constexpr ntfs_duration day(std::chrono::hours(24));
-	constexpr ntfs_duration year(day * days_in_year);
-	constexpr ntfs_duration four_years(day * days_in_four_years);
-	constexpr ntfs_duration century(day * days_in_century);
-	constexpr ntfs_duration four_centuries(day * days_in_four_centuries);
+	// can't use std::chrono::system_clock::duration here, out of fear of integer overflow
+	constexpr std::chrono::duration<std::int64_t, std::ratio<1, 1> > day(std::chrono::hours(24));
```

Comment:
Could you use a local `typedef` inside the function for this?  There's a lot of visual clutter.